src/components/Alert/index.tsx

```diff
@@ -17,67 +18,41 @@ export const alertVariants = [
 type AlertType = typeof alertTypes[number]
 type AlertVariant = typeof alertVariants[number]
 
-const typesColors = (
-  theme: Theme,
-): Record<AlertType, { primary: string; secondary: string }> => ({
-  beta: {
-    primary: theme.colorsDeprecated.beta,
-    secondary: theme.colorsDeprecated.serenade,
-  },
-  info: {
-    primary: theme.colorsDeprecated.info,
-    secondary: theme.colorsDeprecated.zumthor,
-  },
-  success: {
-    primary: theme.colorsDeprecated.success,
-    secondary: theme.colorsDeprecated.foam,
-  },
-  warning: {
-    primary: theme.colorsDeprecated.warning,
-    secondary: theme.colorsDeprecated.pippin,
-  },
-})
+const alertTypeToSentimentMapping: Record<AlertType, Sentiment> = {
+  beta: 'warning',
+  info: 'info',
+  success: 'success',
+  warning: 'danger',
+}
 
-const variants = ({
-  type,
-  theme,
-}: {
-  theme: Theme
-  type: AlertType
-}): Record<AlertVariant, SerializedStyles> => {
-  const primary =
-    typesColors(theme)[type]?.primary || theme.colorsDeprecated.warning
-  const secondary =
-    typesColors(theme)[type]?.secondary || theme.colorsDeprecated.pippin
+const alertStyles = (
+  theme: Theme,
+  type: AlertType,
+): Record<AlertVariant, SerializedStyles> => {
+  const sentiment =
+    theme.colors[alertTypeToSentimentMapping[type]] || theme.colors.warning
```

Comment:
```suggestion
    theme.colors[alertTypeToSentimentMapping[type]] || theme.colors.danger
```

Warning is orange while danger, this way snapshots shouldn't require any update 😛

Reply:
fixed